test/common_methods_invocations.py

```diff
@@ -251,6 +253,7 @@ def method_tests():
         ('clamp', (), (0, 1), 'scalar', (True,)),
         ('clamp', (), (None, 0.5), 'min_scalar', (True,)),
         ('clamp', (), (0.5, None), 'max_scalar', (True,)),
+        ('clamp', (S, S), (), 'max_scalar_kwarg', (True,), (), (), ident, {'max': 1}),
```

Comment:
This is an example test entry I added that we couldn't handle without the changes to test_jit.py and test_autograd.py (the rest are test changes below are just the ident replacement).